pkg/framework/host.go

```diff
@@ -48,8 +49,9 @@ type Host struct {
 	logger     micrologger.Logger
 	filelogger *filelogger.FileLogger
 
+	extClient            *apiextensionsclient.Clientset
 	g8sClient            *versioned.Clientset
-	k8sClient            kubernetes.Interface
+	k8sClient            *kubernetes.Clientset
```

Comment:
I just prefer to have most explicit type possible. That makes it easier to jump around the code and figure things. Also other clients where typed that way.